### DISCUSSION TITLE
1.3.1 works on Primary - but not when .Chia folder moved to new computer

### DISCUSSION THREAD

**Discussion by RedPillAussie** [Chia-Network/chia-blockchain#10879] [Support]
Hi, my primary computer is having hardware issues, still works but intermittent network connection. I've upgraded chia on this computer to v1.3.1 and all working and farming with no issues. 

I also setup a replacement computer with Chia 1.3.1. Copied the primary computers ".CHIA folder", deleted the .CHIA folder on the replacement computer and pasted in the .CHIA folder from the primary computer. Reason is to allow the replacement computers CHIA start up to commence around the same sync level as the primary computer so I don't have to sync all over again with the network.

Started up CHIA on replacement computer (primary computer chia was fully closed down) - yet CHIA on the replacement computer starts all the services but gets stuck on Full Node. 

Operating System ; Window' 10, all up to date with latest patches.
No Security running except windows defender, allowed chia farmer to connect.

![Chia 1 3 1 stuck on FullNode](https://user-images.githubusercontent.com/88886706/159885376-3b10b867-04db-410c-b3a8-b42218fa831b.PNG)

Is there perhaps a different method to moving an existing CHIA farm to another computer without having to start all over again. I've performed this same process in 2021 with earlier versions of CHIA and all worked perfectly and the new computer sync'd up in no time

Perhaps CHIA 1.3.1 wont allow copying over the .CHIA folder or there's a bug? 

Regards 
RedPillAussie

**Comment by loppefaaret** [comment]
Could you check your `debug.log` for ERROR or WARNING from the "full_node" process ? it is located under your users home folder, in `~\.chia\mainnet\log\debug.log` - the full path would then be `C:\Users\<USERNAME>\.chia\mainnet\log\debug.log`
if you are unsure, you can also attach that file to this issue

**Reply by RedPillAussie** [reply]
Attached is a snapshot of the top part of the Debug.log file (_couldn't attach to this discussion thread?_) 

You'll note there are some "does not exist" warnings as I had deleted some older "final destination directories" during my initial setup that I was playing with to learn the whole process about 6+ months ago (deleted directories were SPACE-POOL-4, SPACE-POOL-3, MADMAX, SPACE POOL) 

**WARNING  Directory: \\Agm-nas-130661\chia farm plots\SPACE-POOL-4 does not exist.**
WARNING  Directory: \\Agm-nas-130661\chia farm plots\SPACE-POOL-3 does not exist.
WARNING  Directory: \\Agm-nas-130661\chia farm plots\SPACE-POOL does not exist.
WARNING  Directory: \\Agm-nas-130661\chia farm plots\MADMAX does not exist.

_(If these checks and rechecks of the deleted folders ((SPACE-POOL-4, SPACE-POOL-3, MADMAX, SPACE POOL) are also slowing the process down I'm not sure how to remove these checks / warnings from the original .CHIA folder so their not continually checked etc.... Any advice on how to clean up and tweak CHIA would be appreciated, perhaps for a different discussion thread)_

So from about 5 Months ago, I only have one final destination folder called "**SPACE-POOL-2**" which is where all my plots are located. 

Otherwise, CHIA /SpacePool Works 100% fine and farming well on the **Primary computer**, so nothing wrong with the farm itself. I also have 100% Valid Partials on the primary computer so the timing is well within the limits of below 5secs. _(Perhaps still needs some fine tuning if it keeps checking and sending warnings for the old deleted folder?)_

The only thing I can think of that could possibly cause the new CHIA Farm from not starting the Full Node or sitting there for ages on Full Node, is the limitations of the replacement computer. Attached is a config of the replacement computer I'm using while I have the Primary one repaired.

![Computer Config](https://user-images.githubusercontent.com/88886706/160227007-4c2eba8a-7c07-4f66-bdb5-6777e63a5e79.PNG)
![debug folder](https://user-images.githubusercontent.com/88886706/160227073-aa5d4b24-f743-4181-9254-a4229c056e31.PNG)
![Debug Capture](https://user-images.githubusercontent.com/88886706/160227117-3257ddbf-6821-4737-b617-b6026d2ba67a.PNG)

Hope this helps as I'm bit lost on what to do next as I need to move my current CHIA farm to a replacement computer and not lose Sync.

Kind regards
RedPillAussie

**Comment by loppefaaret** [comment]
The "correct" way to migrate the blockchain data from one machine to another, is to copy the needed data alone.
Copying the entire `~/.chia` folder can lead to some issues.
if you are using version 1.3.X - you need to be aware that you should be using database v2 on all machines.
once that is done, shutdown the client on the machine that you want to take the blockchain database from.
then take a copy of the `~/.chia/mainnet/db` folder
once you got that copy made, you can open up the chia client again on the original machine
take the copy to the new machine, and close down the chia client.
then clean out the existing `~/.chia/mainnet/db` folder, and move in the copy
then open up the new machines chia client
it should pick up from the point of copy, and do an initial sync up to the current tip of the blockchain.

the above logging is not of much use, as it is weeks old, and do have a few entries from the "full_node" service running...
